internal/upgradecheck/http_test.go

```diff
@@ -69,7 +69,7 @@ func TestCheckForUpgrades(t *testing.T) {
 		assert.Equal(t, data.RegistrationToken, "speakFriend")
 		assert.Equal(t, data.BridgeClustersTotal, 2)
 		assert.Equal(t, data.PGOClustersTotal, 2)
-		assert.Equal(t, data.FeatureGatesEnabled, "TablespaceVolumes=true")
+		assert.Equal(t, data.FeatureGatesEnabled, "AutoCreateUserSchema=true,TablespaceVolumes=true")
```

Comment:
👍🏻 This isn't `Contains`, but I don't mind seeing the format. 🌱 We can change the test if it is ever a problem.